File: java/basicsports/src/main/java/org/tcf/basicsports/SportsApp.java
/**
 * SportsApp.java
 * @version 1.0.0 8/31/2001
 * @author Michael P. Redlich
 */

package org.tcf.basicsports;

public class SportsApp {
    /**
     *
     * @param args
     */
	public static void main(String[] args) {
        SportsApp sportsApp = new SportsApp();
        sportsApp.displayHeader();

		Sports nymets = new Baseball("Mets",97,65);
		nymets.display();

        Sports nyjets = new Football("Jets",12,3,1);
		nyjets.display();

        int win = nymets.getWin();
		System.out.println("The number of wins for the " + nymets.getTeam() + " is " + win);
		}

    /**
     *
     */
    void displayHeader() {
        System.out.println();
        System.out.println("*--------------------------------------------------*");
        System.out.println("* Getting Started with Java                        *");
        System.out.println("* Sports Demo Application                          *");
        System.out.println("*--------------------------------------------------*");
        System.out.println();
        }
    }

File: cpp/basicsports/SportsApp.cpp
/**
 * SportsApp.cpp
 * @version 1.0.1 3/23/2015
 * @author Michael P. Redlich
 */

#include <iostream>
#include "Sports.h"
#include "Baseball.h"
#include "Football.h"

void displayHeader(void) {
    cout << endl;
    cout << "*--------------------------------------------------*" << endl;
    cout << "* Getting Started with C++                         *" << endl;
    cout << "* Sports Demo Application                          *" << endl;
    cout << "*--------------------------------------------------*" << endl;
    cout << endl;
    }

int main(int argc,char *argv[]) {
    displayHeader();
    
	Sports *mets = new Baseball("Mets",97,65);
	mets->display();
    
	Sports *jets = new Football("Jets",12,3,1);
	jets->display();
	
    return 0;
	}
